### PR TITLE
Regenerate with latest GIR

### PR DESCRIPTION
**Pull request by sdroege**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,10 +18,11 @@ v1_32_4 = ["v1_32", "pango-sys/v1_32_4"]
 v1_36_7 = ["v1_34", "pango-sys/v1_36_7"]
 v1_34 = ["v1_32_4", "pango-sys/v1_34"]
 v1_31 = ["pango-sys/v1_31"]
+dox = []
```
Comment (sdroege) [inline Cargo.toml]: Is this how it is supposed to be? gtk and gio look like this

Comment (GuillaumeGomez) [reply Cargo.toml]: Not exactly. It needs to trigger the dox feature of `pango-sys` (and of `glib`, I think?).

Comment (EPashkin) [reply Cargo.toml]: Better `dox = ["pango-sys/dox", "glib/dox"]` as in https://github.com/gtk-rs/gio/pull/57

Comment (GuillaumeGomez) [reply Cargo.toml]: So just like I said. :D Thanks for the confirmation @EPashkin!

Comment (sdroege) [reply Cargo.toml]: Why also of glib? For the -sys crate it makes sense because it uses feature-gated API there

Comment (sdroege) [reply Cargo.toml]: And then various crates needs to be fixed, before my regens already :)

Comment (GuillaumeGomez) [reply Cargo.toml]: Yep, sorry. Was doing it veeeeeeeery slowly.

Comment (EPashkin) [reply Cargo.toml]: @GuillaumeGomez Only needed `cairo` crate, not sure what it need be in `dox` as `glib` is optional.

Comment (sdroege) [reply Cargo.toml]: So should each crate's dox feature only depend on the -sys/dox feature?

Comment (EPashkin) [reply Cargo.toml]: IMHO gdk is exception it need include gio, pango, cairo and maybe glib too.

Comment (sdroege) [reply Cargo.toml]: Ok, why? :) Someone please fix it for me or explain to me how it works ;)

Comment (EPashkin) [reply Cargo.toml]: My opinion: if someone try to build docs only for gdk he get all functions for it but only part for dependencies.
Also as gtk will depends only for gdk it help also will be partial.

Comment (GuillaumeGomez) [reply Cargo.toml]: It's in order to overpass the rust's limitations so we can get docs from every functions and not only the ones from the archs/version.

Comment (sdroege) [reply Cargo.toml]: Yes but for that it's enough to do have $crate's dox feature to depend on $crate-sys/dox? Or should every possible dependency be added there?

Comment (GuillaumeGomez) [reply Cargo.toml]: Well, if you don't pass the parameters to your non-sys dependencies, then it won't have the correct rendering (the non-dox one).

Comment (sdroege) [reply Cargo.toml]: Ok, but that's not very ergonomic then. You would need to list each of your dependencies again then for completeness :(

What do you want to have added here?

Comment (GuillaumeGomez) [reply Cargo.toml]: Should be enough, yes.

**Comment by EPashkin** [comment]
👍 after CI passed

File: Cargo.toml (hunk 1 above)
Comment (sdroege) [reply Cargo.toml]: Ok, perfect :)

**Comment by EPashkin** [comment]
As I remember main reason for not including `pangocairo` to `gtk-rs` was patched .gir.
See https://github.com/gtk-rs/gtk/issues/453

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**